AllisonsCatPalaceTDD/FavoritesListController.swift

```diff
@@ -7,8 +7,28 @@
 //
 
 import UIKit
+import RealmSwift
 
-class FavoritesListController: UITableViewController {
+// This should be the protocol for the container
+protocol RealmProtocol {
```

Comment:
should be in it's own file. All the realm dependency injection crap should be in it's own file.

Reply:
done